eclipse-scout-core/src/desktop/outline/pages/js/JsPageHelper.less

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+.outline.tree > .tree-data {
```

Comment:
Remove unnecessary outer rules (`invisible` mixin specifies !important, so it will be applied even if the remaining rule was too unspecific)